src/shipit_uplift/tests/test_coverage.py

```diff
@@ -64,7 +64,7 @@ def test_coverage_summary_by_changeset_impl(coverage_builds):
     # Get code coverage information for a given file at a given changeset
     for file_coverage in coverage_changeset_by_file:
         data = await coverage_for_file_impl.generate(file_coverage['changeset'], file_coverage['path'])
-        assert data == file_coverage['data']
+        assert data['data'] == file_coverage['data']
```

Comment:
Can you also assert the other two values? You might need to add them to `src/shipit_uplift/tests/fixtures/coverage_changeset_by_file.json`.